var/spack/repos/jcsda-emc-bundles/packages/jedi-base-env/package.py

```diff
@@ -27,7 +27,8 @@ class JediBaseEnv(BundlePackage):
     depends_on("blas", type="run")
     depends_on("boost", type="run")
     depends_on("bufr", type="run")
-    depends_on("crtm@v2.4.1-jedi", type="run")
+    # Force users to load manually
+    #depends_on("crtm@v2.4.1-jedi", type="run")
```

Comment:
@climbfuji Correct me if I'm wrong, but we want to keep v2.4.1 in the stack.  It's only v3.0.0 that won't be in the stack until it's settled down a bit (next release probably)

Reply:
I need to correct you, somewhat. Yes, we want to keep v2.4.1 (and 2.4 for Navy) in the stack, but we don't want to load them automatically. This is to avoid adding one more step to the instructions (`module unload crtm ...`). See accompanying spack-stack PR.